A3A/addons/core/Templates/Templates/SWE/templates.hpp

```diff
@@ -31,4 +31,28 @@
         name = "SFP Sweden Winter";
         file = "SFP_AI_SWE_Arctic";
         climate[] = {"arctic"};
+    };
+
+    class FFP_Base : SFP_Base
+    {
+        requiredAddons[] = {"Finnish_Forces_Pack"};        // units, weapons, vehicles
```

Comment:
does this only require FFP or does it also require SFP/CUP? This will overwrite, not append

Reply:
ah, no, it requires SFP and CUP. I'll update it :)